src/index.ts

```diff
@@ -151,3 +151,18 @@ async function compileOas(config: Config) {
     doc: openApiObject,
   };
 }
+
+function skipValidation(validatePaths: Array<string> | Array<RegExp>, ctx: koa.Context) {
+  let dontValidate = !validatePaths.some((path: string | RegExp) => {
+    if (isRegexPattern(path)) {
+      return (path as RegExp).test(ctx.path);
+    } else {
+      return ctx.path.startsWith(path as string);
+    }
+  });
+  return dontValidate;
+}
+
+function isRegexPattern(path: string | RegExp) {
```

Comment:
Can you make this a type guard function so that we don't have to cast to `RegExp` in the above?